### PR TITLE
feat: rename to CTIS results

### PR DESCRIPTION
**Pull request by sgratzl**
**Prerequisites**:

- [x] Unless it is a hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary

rename survey results page

### CODE REVIEW

**Comment by krivard** [comment]
Also the overview page

image

**Comment by sgratzl** [comment]
> Also the overview page

good catch I was just focusing on the survey results page

**Review by krivard — APPROVED** [review]
I'm set, but I'm looping Wichada in to see if there's anything we missed. Wichada you should be okay to just look at the preview link from the netlify comment above

File: src/modes/landing/Landing.svelte [hunk 1]
```diff
@@ -125,9 +125,9 @@
 
     <div class="block">
       <hr />
-      <FancyHeader sub="Pandemic Survey via Facebook" center>Delphi</FancyHeader>
+      <FancyHeader sub="Survey" center>COVID Trends and Impact</FancyHeader>
       <p>
-        The U.S. Pandemic Survey offers insights into public sentiment on:
+        The Trends and Impact Survey (CTIS) offers insights into public sentiment on:
```
Comment (capnrefsmmat) [inline src/modes/landing/Landing.svelte:130]:
```suggestion
        The COVID Trends and Impact Survey (CTIS) offers insights into public sentiment on:
```

**Review by capnrefsmmat — CHANGES_REQUESTED** [review]
One suggested change. Also, where do I go now to update the signal description text so it mentions the name directly?

**Comment by sgratzl** [comment]
> Also, where do I go now to update the signal description text so it mentions the name directly?

so far still at https://docs.google.com/document/d/1J36nedP6eufeRE3TbyT_BK-ofE5d18kLif2UmORkYDU/edit#

**Comment by sgratzl** [comment]
and https://docs.google.com/document/d/1llv6xh8jMlmVv7WpyDSv4VgUpFAZQ6QjFUeRxxOinmk/edit#heading=h.oxebzdy7nl8w

File: src/modes/landing/Landing.svelte [hunk 2]
```diff
@@ -125,9 +125,9 @@
 
     <div class="block">
       <hr />
-      <FancyHeader sub="Pandemic Survey via Facebook" center>Delphi</FancyHeader>
+      <FancyHeader sub="Survey" center>COVID Trends and Impact</FancyHeader>
       <p>
-        The U.S. Pandemic Survey offers insights into public sentiment on:
+        The COVID Trends and Impact Survey (CTIS) offers insights into public sentiment on:
```
Comment (capnrefsmmat) [inline src/modes/landing/Landing.svelte:130]:
```suggestion
        The COVID-19 Trends and Impact Survey (CTIS) offers insights into public sentiment on:
```

File: src/modes/survey-results/SurveyResults.svelte [hunk 3]
```diff
@@ -51,7 +51,7 @@
 <div class="root">
   <RegionDatePicker sensor={refSensor} items={filteredInfos} defaultItem={nationInfo}>
     <div class="grid-3-11 mobile-header-line" slot="title">
-      <h2>Delphi Survey <span>Results</span></h2>
+      <h2>COVID Trends and Impact Survey (CTIS) <span>Results</span></h2>
```
Comment (capnrefsmmat) [inline src/modes/survey-results/SurveyResults.svelte:54]:
```suggestion
      <h2>COVID-19 Trends and Impact Survey (CTIS) <span>Results</span></h2>
```

**Review by capnrefsmmat — CHANGES_REQUESTED** [review]
Turns out we want to include -19 in the name

**Comment by capnrefsmmat** [comment]
I've also suggested a change in the dashboard config doc (mentioning the name)

**Review by capnrefsmmat — APPROVED** [review]
Looks good

**Pull request merged**